tests/functional/desy/fixtures/ftp_server/DESY/desy_collection_records.xml

```diff
@@ -1,926 +1,363 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <collection xmlns="http://www.loc.gov/MARC21/slim">
-    <record>
-        <controlfield tag="001">111111</controlfield>
```

Comment:
Why did you get rid of the control fields?

Reply:
I took real batchupload records. Those were inserts, so didn't have any controlfield yield.

Reply:
okok, I'll work around that 👍